packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -239,7 +262,8 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 	public ngOnInit(): void {
 		this.logger.debug(componentName + ": component initialized");
 		this.displayedColumns = [];
-		if (this.multiselect) {
+		if (this.isMultiSelectEnabled) {
+			console.log("------- isMultiSelectEnabled ???", this.isMultiSelectEnabled);
```

Comment:
Don't forget to remove the `console.log` ;)

Reply:
Oups 😋